tests/unit/ExtensionRequest.spec.ts

```diff
@@ -36,16 +36,13 @@ describe('ExtensionRequest', () => {
     expect(rows.at(1).find('.col-sm-3').text()).toBe('AGM Year')
     expect(rows.at(1).find('.col-sm-9 .v-text-field').exists()).toBe(true)
 
-    expect(rows.at(2).find('.col-sm-3').text()).toBe('Previous AGM date or a reference date')
-    expect(rows.at(2).find('.col-sm-9').findComponent(DatePicker).exists()).toBe(true)
-
```

Comment:
I'm not 100% sure why this worked originally since when the component loads with empty values, this should not be rendered. Even in dev when the extension request first loads this row is not present in the DOM.